skops/card/_model_card.py

```diff
@@ -373,6 +374,34 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_feature_importances(self, feature_importances, data) -> "Card":
```

Comment:
I wonder about the naming. The method uses "feature_importances", so intuitively, I'd think I can pass something like `est = RandomForestClassifier().fit(X, y); card.add_feature_importances(est.feature_importances_)` but this won't work. If it only works with permutation importances, should the method be renamed accordingly?

Alternatively, we could try to support passing `est.feature_importances_`, as it would be quite convenient, though the meaning of the plot would change.

Reply:
I renamed it to be permutation importance specifically.